libs/sdk-core/src/breez_services.rs

```diff
@@ -81,21 +81,35 @@ pub trait EventListener: Send + Sync {
 #[allow(clippy::large_enum_variant)]
 pub enum BreezEvent {
     /// Indicates that a new block has just been found
-    NewBlock { block: u32 },
+    NewBlock {
+        block: u32,
+    },
     /// Indicates that a new invoice has just been paid
-    InvoicePaid { details: InvoicePaidDetails },
+    InvoicePaid {
+        details: InvoicePaidDetails,
+    },
     /// Indicates that the local SDK state has just been sync-ed with the remote components
     Synced,
     /// Indicates that an outgoing payment has been completed successfully
-    PaymentSucceed { details: Payment },
+    PaymentSucceed {
+        details: Payment,
+    },
     /// Indicates that an outgoing payment has been failed to complete
-    PaymentFailed { details: PaymentFailedData },
+    PaymentFailed {
+        details: PaymentFailedData,
+    },
     /// Indicates that the backup process has just started
     BackupStarted,
     /// Indicates that the backup process has just finished successfully
     BackupSucceeded,
     /// Indicates that the backup process has just failed
-    BackupFailed { details: BackupFailedData },
+    BackupFailed {
+        details: BackupFailedData,
+    },
+    // Indicates that we have just updated the swap associated information
```

Comment:
```suggestion
    /// Indicates that this swap's status was just updated
```

This more strongly indicates that mainly the status has changed, and not only just some of the fields.

Reply:
I combined your suggestion in the comment as we we emit events not only when the status has changed.

Reply:
I see, then I misunderstood it. I thought the event is only emitted on status change.

Reply:
May I ask what's the use case for that? Would a user want to get an event for something other than a status change?

Reply:
Sure, among other things we update the swap chain information such as refund tx id (after refund).
We also update the swap open channel fees from time to time.

Reply:
Makes sense, thanks.